ci-operator/config/openshift/loki/openshift-loki-release-6.0.yaml

```diff
@@ -0,0 +1,161 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
+  loki-tag:
```

Comment:
I think you don't need this rename when `loki` and `promtail` are removed from `images`.